frontend/packages/dev-console/src/components/monitoring/queries.ts

```diff
@@ -22,6 +22,10 @@ export const metricsQuery = {
   PODS_BY_FILESYSTEM: 'Filesystem Usage',
   PODS_BY_NETWORK_IN: 'Receive Bandwidth',
   PODS_BY_NETWORK_OUT: 'Transmit Bandwidth',
+  RATE_OF_RECEIVED_PACKETS: 'Rate of Received Packets',
```

Comment:
is this `Rate of Received Packets` should this be `Rate Of Received Packets`? check with UXD one

Reply:
we have the same title on the dashboard. 
cc: @serenamarie125  @beaumorley